packages/waku/src/lib/plugins/vite-plugin-nonjs-resolve.ts

```diff
@@ -7,17 +7,16 @@ export function nonjsResolvePlugin(): Plugin {
   return {
     name: 'nonjs-resolve-plugin',
     async resolveId(id, importer, options) {
-      if (!id.endsWith('.js')) {
-        return id;
```

Comment:
We shouldn't resolve it because other plugins can't resolve it. We should return `undefined`.